src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.cs

```diff
@@ -821,8 +821,8 @@ private static string NormalizeCultureName(string name, out bool isNeutralName)
             if (GlobalizationMode.Hybrid && !string.IsNullOrEmpty(culture._sName))
             {
                 culture = JSLoadCultureInfoFromBrowser(culture._sName, culture);
+                culture.JSInitLocaleInfo();
             }
```

Comment:
```suggestion
            }
#if !NATIVEAOT // TODO-LLVM: <issue link>.
            culture.JSInitLocaleInfo();
#endif
```
I've looked into this a bit, and it is from this change: https://github.com/dotnet/runtime/pull/99956.

Basically, even non-hybrid globalization now depends on some JS code. Could you open a new issue about this and link to it here?